client/src/components/auth/auth.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
```

Comment:
i think if you add this font in a global css file may be better to keep the website with the same font